src/core.jl

```diff
@@ -61,66 +61,69 @@ end
 - `kernel::function`: optional distance kernel; one of (`bubbleKernel, gaussianKernel`)
             default is `gaussianKernel`
 - `r`: optional training radius.
-       If r is not specified, it defaults to √(xdim^2 + ydim^2) / 2
+       If r is not specified, it defaults to (xdim+ydim)/3
+- `rFinal`: target radius at the last epoch, defaults to 0.1
 Training data must be convertable to Array{Float34,2} with `convert()`.
 Training samples are row-wise; one sample per row. An alternative kernel function
 can be provided to modify the distance-dependent training. The function must fit
 to the signature fun(x, r) where x is an arbitrary distance and r is a parameter
 controlling the function and the return value is between 0.0 and 1.0.
 """
-function trainGigaSOM(som::Som, train::DataFrame; kernelFun::Function = gaussianKernel,
-                    rStart = 0.0, epochs = 10, decay = "linear")
+function trainGigaSOM(som::Som, train::DataFrame;
+                      kernelFun::Function = gaussianKernel,
+                      knnTreeFun = BruteTree,
+                      rStart = 0.0, decay = "linear", rFinal=0.1,
+                      epochs = 10)
 
     train = convertTrainingData(train)
 
     # set default radius:
+
     if rStart == 0.0
-        rStart = √(som.xdim^2 + som.ydim^2) / 2
+        rStart = √(som.xdim^2 + som.ydim^2) / 3
```

Comment:
/ 2 was from the original som implementation, in my view it is a good starting radius if not specified otherwise

Reply:
@oHunewald, so is `/2` more appropriate than `/3`?

Reply:
I would like to keep /2 as this has been our default radius all the time and gives reasonable results. Otherwise one can pass different start radius as parameter